src/app/shared/types/module.types.ts

```diff
@@ -1,2 +1,4 @@
 // tslint:disable: no-any
 export type Module = any;
```

Comment:
this should be removed if were using modules type instead of it

Reply:
No we're using this also